includes/MidiEventIn.h

```diff
@@ -3,6 +3,7 @@
 
 // --------------------------
 #include <map>
+#include <string>
```

Comment:
Really we don't need this `include`.

Reply:
Building with `cmake --build .` fails for me without this. See the build logs in the PR desc. Am I just doing something wrong here?